cnames_active.js

```diff
@@ -2208,7 +2209,7 @@ var cnames_active = {
   "vormulir": "bramaudi.github.io/vormulir",
   "vorpal": "dthree.github.io/vorpal", // noCF? (don´t add this in a new PR)
   "voxelcss": "jhwohlgemuth.github.io/voxelcss",
-  "vrv": "cloudflare-to-cloudflare.readthedocs.io",
+  "vrv": "hosting.gitbook.com",
```

Comment:
```suggestion
  "vrv": "cloudflare-to-cloudflare.readthedocs.io",
```
Please revert this change -- if you wish to update this target, please create a separate PR.